app/routers/users.py

```diff
@@ -221,9 +225,37 @@ async def get_user_accounts(id: str):  # pragma: no cover
     "/{id}/disable",
     dependencies=[Depends(authentication_required)],
     response_model=UserRead,
+    status_code=status.HTTP_201_CREATED,
 )
-async def disable_user(id: str):  # pragma: no cover
-    pass  # not yet implemented
+async def disable_user(
+    id: str,
+    user_repo: UserRepository,
+    auth_context: CurrentAuthContext,
+    user: Annotated[User, Depends(fetch_user_or_404)],
+):
+    """
+    This endpoint allows an OPERATOR to disable a user.
+    A user cannot disable itself.
+    """
+    if auth_context.account.type != AccountType.OPERATIONS:
```

Comment:
We already have this [https://github.com/softwareone-platform/ffc-finops-operations/blob/main/app/auth/auth.py#L143](https://github.com/softwareone-platform/ffc-finops-operations/blob/main/app/auth/auth.py#L143).